tests/commands/test_set.py

```diff
@@ -94,11 +114,10 @@ def test_set_with_dot_recursive(repo: Repo, set_values: list[str]) -> None:
     Test that when `onyo set KEY=VALUE .` is called from the repository root,
```

Comment:
Out of scope of the current PR, but if you end up updating this PR, perhaps you can fold these in:
- I think this docstring is wrong. It should call from `cwd`, not repo root. Buuuuuuuut
- I don't think this test is necessary. `.` is a dir, just like any other. So this can be removed.

Reply:
It is logically/functionally equivalent, but just if it works. But I think, especially since we are still changing the way argparser works, it is worth testing to make sure that this gets correctly interpreted.

Also, in the past we had cases, were the onyo root was not part of the `Repo.dirs`, which should be tested else, but still was found through a test like this. So, as soon as we work on fixing the Problems with working from a subdir, I would rather argument that similar additional tests should be added.

Reply:
> It is logically/functionally equivalent, but just if it works.

`.` is a valid dir for a POSIX filesystem. So I'm missing the situation where it wouldn't work.

> Also, in the past we had cases, were the onyo root was not part of the Repo.dirs, which should be tested else, but still was found through a test like this. 

If there isn't already a test for `Repo.dirs` that checks for this, then it should be added there.

Reply:
Thanks, you are right.

That would then be here:
https://github.com/psyinfra/onyo/blob/main/tests/lib/test_Repo.py#L253

I will think about it and either add it to the current tests, or do it together with #253 or #251.

Reply:
I added a new issue describing the need for the `tests/lib/*.py` updates and will add/solve this when I overhaul them.